docs/install.rst

```diff
@@ -106,16 +106,20 @@ Python package dependencies
 - seaborn >=0.9.0
 - joblib >=0.11
 
-with optional [torch] dependencies
+with optional [torch] dependencies,
 
 - torch >=1.1.0
 - tqdm
 
-and optional [multiviewica] dependencies
+optional [multiviewica] dependencies,
 
 - python-picard >= 0.4
 - multiviewica >= 0.0.1
```

Comment:
picky, but can you remove the space between >= and the version so picard and multiviewica match the formatting of the other dependencies listed above?